src/domains/home/HomeScreen.tsx

```diff
@@ -1,9 +1,29 @@
-import React, { FC } from 'react';
+import React, { FC, useCallback } from 'react';
+import { View } from 'react-native';
+import { useDispatch } from 'react-redux';
+import { Button } from 'src/components/Button';
 import { Screen } from 'src/components/Screen';
 import { Text } from 'src/components/Text';
+import { AppDispatch } from 'src/state';
+import { logout } from 'src/state/authSlice';
 
-export const HomeScreen: FC = () => (
-  <Screen variant="centered">
-    <Text fontWeight="black">Hello World</Text>
-  </Screen>
-);
+export const HomeScreen: FC = () => {
+  const dispatch = useDispatch<AppDispatch>();
+  const onPressLogout = useCallback(() => {
+    dispatch(logout());
+  }, [dispatch]);
```

Comment:
i suspect technically `logout` should be a dependency but i doubt it matters

Reply:
Logout is declared outside of the of scope of the component, so the dep wouldn't be neceessary